react-components/src/components/Widgets/WindowWidget.tsx

```diff
@@ -115,6 +117,7 @@ export const WindowWidget = ({
           onResize={handleResize}>
           <Widget>
             <Widget.Header title={title} type={type} header={header} subtitle={subtitle}>
+              {openInNewTab !== undefined && <div>{openInNewTab}</div>}
```

Comment:
Is this `div` necessary? I just think the fewer components we need to squeeze in here and there, the better :sweat_smile: